packages/react-examples/src/react-components/Migrations/utils.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Source } from '@storybook/addon-docs/blocks';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const codeComparisonUseStyles = makeStyles({
+  root: {
+    display: 'flex',
+    justifyContent: 'space-between',
+
+    '> *': {
+      width: '100%',
+    },
+
+    '> :not(:last-child)': {
+      marginRight: '5%',
+    },
+  },
+});
+
+export const CodeComparison = (props: { children: React.ReactElement[] }) => {
+  const { children } = props;
+  const classes = codeComparisonUseStyles();
+
+  return <div className={classes.root}>{children}</div>;
+};
+
+export const CodeLanguages: { [key: string]: string } = {
```

Comment:
nit: do you need to type this, I don't see any usage of the type anywhere. If you do need this, it might be cleaner to use:

```suggestion
export const CodeLanguages: Record<string, string> = {
```